tests/installation/bootloader_s390.pm

```diff
@@ -314,7 +314,11 @@ sub run {
     }
 
     # format DASD before installation by default
-    format_dasd                if (check_var('FORMAT_DASD', 'pre_install'));
+    if (get_var('WORKAROUND_BUGS') =~ 'bsc1178790') {
+        record_soft_failure('bsc#1178790 - dasd_configure: Error: Could not write file /sys/bus/ccw/drivers/dasd-eckd/0.0.0150/online: Resource temporarily unavailable');
+    } elsif (check_var('FORMAT_DASD', 'pre_install')) {
+        format_dasd;
+    }
```

Comment:
Maybe I don't see it but where do you detect the actual error condition? Or do you not? Should be possible to *try* to format and if that does not work then look for the error in the output or logs and record a soft failure then. An unconditional soft-failure is not a good idea due to causing false positives which annoy people soon and make them distrust openQA tests.

Reply:
Hi,

Josef already tried that: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/installation/bootloader_s390.pm#L272

The problem is that the disk becomes unresponsive. The only way to recover it is to restart the installer.

1. I can think of restarting the installation process if the bug is detected...
2. Or, creating a separate scenario with the only purpose of reproducing that bug, so I can workaround failing tests by setting `FORMAT_DASD=` (empty) and letting the dedicated scenario fail hard red.

I will go for 2., unless I received feedback from you telling me that the unconditional workaround provided in this PR is better than 2.

Reply:
ok, if it's that hard then I would appreciate if you just write a comment explaining so in the source code, then it would be ok.

Both options you mentioned are ok. I would assume that restarting the installation is doable and would be cool and could actually be easier than option 2 because it should be just the same function you are in already. But as I said, all three ways are ok for me.